src/main/java/com/cburch/logisim/fpga/data/MapComponent.java

```diff
@@ -678,7 +681,7 @@ public String getDisplayString(int pin) {
 
   public void getMapElement(Element Map) throws DOMException {
     if (!hasMap()) return;
-    Map.setAttribute(MAP_KEY, getDisplayString(-1));
+    Map.setAttribute(MAP_KEY, getDisplayString(ONLY_IO_MAP_NAME));
```

Comment:
Formerly `getDisplayString()` was given `-1`. Not it is given `ONLY_IO_MAP_NAME` but that's `-2`. Is that correct?

Reply:
Yes, it should be like this, otherwise it get the display name with it like "-> Unconnected"